### ISSUE TITLE
Protonmail-bridge not appeared in search inspite being merged !

### ISSUE THREAD

**Issue by Nokia808** [flathub/ch.protonmail.protonmail-bridge#2]
Hi. I noticed that "protonmail-bridge" already merged, but till now not appearing in search result ! Please reveiw if this due to an error (bug) or not.

https://github.com/flathub/ch.protonmail.protonmail-bridge

Thank you.

**Comment by AsciiWolf** [comment]
It show in GNOME Software, but with no icon. So it seems that there is some issue with the application icon.

**Comment by Nokia808** [comment]
@AsciiWolf 
Hi. Do you modified something ?? Yes, I conform that it is now - AND ONLY NOW NOT BEFORE - I can see it in software cneter !!!! Icon not displayed but it existing & I can see it & brows it & see screenshot ...

But it is still NOT APPEARING in the search of the following link:
https://www.flathub.org/home
try search engine using "proton" , "protonmail" , "protonmail-bridge" & you will get zero results ! I tried even just "mail" & just "bridge" but never appeared in result list !

**Comment by A6GibKm** [comment]
```$ desktop-file-validate test.desktop 

test.desktop: hint: value item "Security" in key "Categories" in group "Desktop Entry" can be extended with another category among the following categories: Settings, or System
test.desktop: hint: value "Utility;Security;Network;Email" for key "Categories" in group "Desktop Entry" contains more than one main category; application might appear more than once in the application menu```

switching to `Categories=Office;Email` fixes the desktop file but I don't know if it fixes the app not showing up in flathub. Is there a way to debug this locally?

**Comment by A6GibKm** [comment]
Should be fixed by #3

**Comment by bilelmoussaoui** [comment]
@A6GibKm thanks for looking into this. I have published the latest build & the app indeed appears on Flathub now https://flathub.org/apps/details/ch.protonmail.protonmail-bridge
